File: version-app/src/main/java/io/ehdev/conrad/app/service/version/model/VersionSearchModel.java
package io.ehdev.conrad.app.service.version.model;

import java.util.List;

public class VersionSearchModel {
    List<String> commits;

    public VersionSearchModel(List<String> commits) {
        this.commits = commits;
    }

    public VersionSearchModel() {
    }

    public List<String> getCommits() {
        return commits;
    }

    public void setCommits(List<String> commits) {
        this.commits = commits;
    }
}

File: api-model/src/main/kotlin/io/ehdev/conrad/model/version/VersionSearchModel.kt
package io.ehdev.conrad.model.version

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.constraints.NotNull

open class VersionSearchModel {

    @JsonCreator
    constructor(@JsonProperty("commits") commits: List<String>) {
        this.commits = commits
    }

    @NotNull
    var commits: List<String>;
}
